services/core/SQLAggregateHistorian/config.mysql

```diff
@@ -13,36 +13,31 @@
     # configuration from mysql historian - END
```

Comment:
Should only be one config file inside the SQLAggregateHistorian directory.  Note the configuration should be setup to use one or the other and have directions in the readme to how to change it.

Reply:
Done